nodedb.go

```diff
@@ -222,6 +251,10 @@ func (ndb *nodeDB) deleteOrphans(version int64) {
 	// Will be zero if there is no previous version.
 	predecessor := ndb.getPreviousVersion(version)
 
+	ndb.deleteOrphansWithPredecessor(version, predecessor, nil)
+}
+
+func (ndb *nodeDB) deleteOrphansWithPredecessor(version, predecessor int64, unsavedOrphans *map[string]int64) {
```

Comment:
what is the purpose of this function w/ the unsavedOrphans arg?  It never gets called?